pages/common/topydo.md

```diff
@@ -0,0 +1,28 @@
+# topydo
+
+> A powerful todo list application using the todo.txt format.
```

Comment:
Perhaps this will read better?
```suggestion
> A todo list application that uses the todo.txt format.
```

Generally we try to avoid subjective terms. Is there another adjective to describe it with (e.g. advanced perhaps?)

Reply:
I copied this sentence from [github/topydo](https://github.com/topydo/topydo). I think advanced does describe it much better.